### ISSUE TITLE
mdMenu: null-pointer exception on mouseover

### ISSUE THREAD

**Issue by Frank3K** [angular/material#5604]
When there are no nested menus, a null-pointer exception is logged when doing a mouseover on an `md-menu-item`.

```
TypeError: Cannot read property 'open' of null
    at angular-material.min.js:13
    at angular.js:17788
    at e (angular.js:5498)
    at angular.js:5775
```

[Demo](http://codepen.io/anon/pen/XmxjXY) showing the issue.

I think the issue is introduced in ead4d022ff5eed664150fce58e4e39cd59942cfd.

Issue is in the [menuController](https://github.com/angular/material/blob/master/src/components/menu/js/menuController.js#L71).

**Comment by DevVersion** [comment]
Its now fixed from @rschmukler  6bf98aa6325bf30b10fa0d586820bd0d0623e933, Please close ;)

**Comment by Frank3K** [comment]
Indeed closed in 6bf98aa.
